src/conversion/DdsDynamicData2OpcUaConversions.cxx

```diff
@@ -772,14 +775,14 @@ void dds_dynamic_data_member_to_dds_dynamic_data_variant(
                 data.value<uint32_t>(member_name));
     } break;
     case dds::core::xtypes::TypeKind::INT_64_TYPE: {
-        variant.value<rti::core::int64>(
```

Comment:
Did you speak to Alex about this change?

Reply:
It seems that set_values() and get_values() are not defined for int64_t and uint64_t, which is what rti::core::int64 maps now. We'll see how to address that.